src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,93 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
+  height: 100%;
+  box-sizing: border-box;
+  padding: 0 $iui-sm;
+  overflow-x: hidden;
+  overflow-y: overlay;
+  resize: horizontal;
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-right: 1px solid t(iui-color-background-5);
+  }
+
+  // #region Animating expanding / collapsing of the submenu
```

Comment:
Better use `@mixin iui-transition-group` and leave animation to React. You only need to add `width` to mixin `transition` or override it.

Reply:
Ok, I think I understand what you are suggesting.  I've made the change in `mixins.scss` and `side-navigation.scss`.